featurebyte/models/observation_table.py

```diff
@@ -126,6 +126,10 @@ class ObservationTableModel(MaterializedTableModel):
         The id of the context that the observation table is associated with
     use_case_ids: Optional[List[PydanticObjectId]]
         The ids of the use cases that the observation table is associated with
+    purpose: Optional[Purpose]
+        THe purpose of the observation table, for example preview, eda, training, etc.
```

Comment:
@vikua A small typo here: THe => The

Reply:
@vikua @gee-senbong the purpose parameter value should correspond to one of accepted values
https://github.com/featurebyte/featurebyte/blob/438c291b32cf561d7b76a49de85f7ddf949a8d85/featurebyte/models/observation_table.py#L107